gatsby-config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Default Starter`,
+    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
+    author: `@gatsbyjs`,
+  },
+  plugins: [
+    `gatsby-plugin-react-helmet`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `images`,
+        path: `${__dirname}/src/images`,
```

Comment:
This folder hasn't been included inside of the repo so the build fails. Can you please add a .gitkeep file to /src/images. Other then that it's all good.

Reply:
Done!